scopes/react/react/jest/transformer.js

```diff
@@ -1,8 +1,26 @@
 /* eslint-disable global-require */
 const { transform } = require('@babel/core');
 
+const hasJsxRuntime = (() => {
```

Comment:
This exact function appears in this PR multiple times.
In this case, you should extract it to its own component and use it from there.
Also, I'm not sure this function is necessary at all. 
I Will explaining further in the below comments

Reply:
this function might be only relevant for the `process.env.DISABLE_NEW_JSX_TRANSFORM` in this case extract it to a central location

Reply:
in case you change this funciton to only check the env variable, you should change its name, as it's not hasJsxRuntime any more, more like `useNewJsxTransform`